beacon_node/client/src/builder.rs

```diff
@@ -525,6 +511,12 @@ where
             ClientGenesis::FromStore => builder.resume_from_db().map(|v| (v, None))?,
         };
 
+        let beacon_chain_builder = if let Some(trusted_setup) = config.trusted_setup {
+            beacon_chain_builder.trusted_setup(trusted_setup)
+        } else {
+            beacon_chain_builder
+        };
```

Comment:
It would be nicer if we don't have to do multiple variable shadowing here, but would probably require changing our builder methods to use mutable references to `self` instead of taking ownership (`self`). I'll leave it out for now to retain the scope of this PR.

e.g. 
```rust
    pub fn trusted_setup(&mut self, trusted_setup: TrustedSetup) -> &mut Self {
        self.trusted_setup = Some(trusted_setup);
        self
    }
```